handlers/mparticle-api/src/routers/baton.ts

```diff
@@ -0,0 +1,134 @@
+import { v4 as uuidv4 } from 'uuid';
+import type { DataSubjectRequestState } from '../../interfaces/data-subject-request-state';
+import { DataSubjectRequestStatus } from '../../interfaces/data-subject-request-state';
+import type { DataSubjectRequestSubmission } from '../../interfaces/data-subject-request-submission';
+import {
+	getStatusOfDataSubjectRequest,
+	submitDataSubjectRequest,
+} from '../apis/data-subject-requests';
+import { setUserAttributesForRightToErasureRequest } from '../apis/events';
+import { getEnv } from '../utils/config';
+
+// https://github.com/guardian/baton/blob/1037c63c9bd782aed514bf6aaa38a54dabf699eb/README.md
+interface BatonRerEventRequestBase {
+	requestType: 'RER';
+}
+
+interface BatonRerEventInitiateRequest extends BatonRerEventRequestBase {
```

Comment:
optional: easy to look at the SAR instead of the RER one (like I just did!) so another link might help
```suggestion
// data structure https://github.com/guardian/baton/blob/1037c63c9bd782aed514bf6aaa38a54dabf699eb/README.md#rer-initiate-call
interface BatonRerEventInitiateRequest extends BatonRerEventRequestBase {
```

Reply:
Improved @johnduffell , thank you for the suggestion 👍